### PR TITLE
Removed newline-adding logic from mimic

### LINKED ISSUE #788: Stop adding newlines between cl mimic bundles
This is totally a nit, but I find it annoying that when you cl mimic, it adds a newline in the worksheet markdown between every bundle.  I usually try to put things in tables to control what columns show up on the web interface, but the newlines mess this up and I have to go in and delete them every time.

### PR DESCRIPTION
**Pull request by bkgoksel**
Fixes #788 
Removing the newline adding logic completely. 

If we find that this breaks some workflows, this commit can be reverted as it is an isolated change.

### CODE REVIEW

File: codalab/lib/bundle_util.py [hunk 1]
```diff
@@ -242,7 +242,6 @@ def newline():
         new_bundle_uuids_added = set()
 
         # Whether there were items that we didn't include in the prelude (in which case we want to put '')
```
Comment (percyliang) [inline codalab/lib/bundle_util.py]: Remove comment too

Comment (percyliang) [reply codalab/lib/bundle_util.py]: Also, can we safely delete `newline()`?

**Review by percyliang — APPROVED** [review]
LGTM modulo cleanup

**Comment by fabeschan** [comment]
(You can ignore the failing travis CI runs)

**Pull request merged**